python/shotgun_menus/entity_field_menu.py

```diff
@@ -110,7 +110,9 @@ def __init__(self, sg_entity_type, parent=None, bg_task_manager=None, project_id
                 start_processing=True
             )
 
-            shotgun_globals.register_bg_task_manager(self._task_manager)
+            # removing this for now... these registrations were causing RV to crash on exit.
+            # are they necessary?
+            # shotgun_globals.register_bg_task_manager(self._task_manager)
```

Comment:
I've been getting crashes on exit with `tk-multi-demos`. I think it only occurs when a demo is loaded that uses a task manager. I can double check and test what happens if one isn't registered.